common/audio_desktop.go

```diff
@@ -0,0 +1,123 @@
+//+build !windows,!netgo,!android
```

Comment:
I guess in the future (as in: before merging), the `engo.io/audio` API would be such that we don't need an `audio_desktop.go` file here, but just one `audio.go` file?